clang/lib/Format/UnwrappedLineParser.cpp

```diff
@@ -5091,8 +5091,10 @@ UnwrappedLineParser::parseMacroCall() {
 void UnwrappedLineParser::pushToken(FormatToken *Tok) {
   Line->Tokens.push_back(UnwrappedLineNode(Tok));
   if (MustBreakBeforeNextToken) {
```

Comment:
Maybe `MustBreakBeforeNextToken` should be renamed.

Reply:
We can do that in a follow-up patch. Any suggestion?

Reply:
Couldn't we just reuse `FirstAfterPPDirectiveLine`?

Reply:
I renamed it to [`AtEndOfPPLine`](https://github.com/owenca/llvm-project/blob/a6ccda28f7569c1a03620d7520de7cfadc11f4a5/clang/lib/Format/UnwrappedLineParser.h#L304) instead.